webpack.config.js

```diff
@@ -83,6 +83,9 @@ const devSiteConfig = (env = {}, argv = { p: false }) => ({
   ],
   resolve: {
     extensions: ['.jst'],
+    alias: {
+      react: path.resolve('./node_modules/react'),
```

Comment:
why do we need alias for React ..??

Reply:
https://jira2.cerner.com/browse/UXPLATFORM-6974?focusedCommentId=11435950&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-11435950